flow-report/src/wrappers/category-score.tsx

```diff
@@ -9,17 +9,21 @@ import {useEffect, useLayoutEffect, useRef} from 'preact/hooks';
 
 import {useReportRenderer} from './report-renderer';
 
-export const Gauge: FunctionComponent<{category: LH.ReportResult.Category, href: string}> =
-({category, href}) => {
+export const CategoryScore: FunctionComponent<{
+  category: LH.ReportResult.Category,
+  href: string,
+}> = ({category, href}) => {
   const {categoryRenderer} = useReportRenderer();
   const ref = useRef<HTMLDivElement>(null);
 
   useLayoutEffect(() => {
     const el = categoryRenderer.renderScoreGauge(category, {});
 
     // Category label is displayed in the navigation header.
-    const label = el.querySelector('.lh-gauge__label');
+    const label = el.querySelector('.lh-gauge__label,.lh-ratio__label');
     if (label) label.remove();
+    const bg = el.querySelector('.lh-ratio__background');
```

Comment:
comment for this one? 

my expectation is the summary doesn't have the background, but I still expect individual reports to have the background (but those are using the report renderer wrapper and not this component)